### ISSUE TITLE
Format specifier %c is useless

### ISSUE THREAD

**Issue by njhanley** [naelstrof/slop#126]
The %c format specifier is currently unusable because slop exits on cancellation without printing formatted output. In practice cancellation is more easily detected by checking slop's exit status or the lack of output. I propose removing this specifier.

**Comment by lukakostic** [comment]
Agreed,  i thought of using %c to detect if cancelled with formatted output, but then i see it prints nothing when -q. So no point in %c
